gulpfile.js

```diff
@@ -102,6 +102,12 @@ gulp.task('soy', function() {
 		.pipe(gulp.dest('packages/metal-soy/test'));
 });
 
+gulp.task('soy:isomorphic', function() {
```

Comment:
Do we need gulp for this? Since we're invoking it from `npm run` can't we simply call `metalsoy`?

Reply:
That's what I did at first, unfortunately this repo still uses an older version of `metal-tools-soy`, and updating it to the version that included the cli caused some issues.

Once this is merged I'd like to update it, resolve the issues, and remove whatever gulp tasks we can.

Reply:
fair enough!